console-extensions.json

```diff
@@ -1,54 +1,26 @@
 [
-  {
+    {
     "type": "console.page/route",
     "properties": {
       "exact": true,
       "path": ["/pipelines-overview/ns/:ns", "/pipelines-overview/all-namespaces"],
-      "component": { "$codeRef": "ExamplePage" }
+      "component": {
+        "$codeRef": "pipelinesComponent.PipelinesOverviewPage"
+      }
     }
   },
   {
     "type": "console.navigation/href",
     "properties": {
-      "id": "pipeline-overview",
+      "id": "pipelines-overview",
       "perspective": "admin",
       "section": "pipelines",
       "insertBefore": "pipelines",
-      "name": "%Overview%",
+      "name": "%plugin__pipeline-console-plugin~Overview%",
```

Comment:
No need to define the ns for the i18n. As it is a single plugin and default ns set as `plugin__pipeline-console-plugin`